inc/blocks/ab-test/register.php

```diff
@@ -113,6 +113,17 @@ function render_block( array $attributes, ?string $inner_content = '' ) : string
 		$winner = $results['winner'] ?? false;
 	}
 
+	if ( is_preview() ) {
+		return sprintf(
+			'%s<div class="ab-test-xb-preview %s" data-client-id="%s"><div class="ab-test-xb-preview__tabs" data-client-tabs="%s"></div><div class="ab-test-xb-preview__content" data-client-content="%s"></div></div>', 
+			$inner_content,
+			$class_name, 
+			$client_id,
+			$client_id,
+			$client_id
+		);
+	}
```

Comment:
I have started this and I wanted to have the tabs and the content identifiable as to which tabs are with which content. Is there a better way to go around this that I may not have to repeat the `$client_id` multiple times?

Reply:
The value for the client ID is already available on the outer div, I'm not sure what the benefit in repeating it would be?

To answer your question though take a look at the sprintf docs: https://www.php.net/manual/en/function.sprintf.php

You can reuse arguments by using the `%<arg position>$s` syntax, so for the first argument you pass `%s` would become `%1$s`, then the second would be `%2$s`. When used that way you can reference them by number as often as you want.

Reply:
Will rework this and remove the items if they not needed any longer

Reply:
To explain a bit further you'd have an initial loop to go through all AB test blocks on the page from where you can get the current block and the client ID, then you'd get the templates associated with it and loop over those, adding the tabs and click handlers.